src/test/ui/malformed/malformed-derive-entry.stderr

```diff
@@ -14,7 +14,7 @@ error: malformed `derive` attribute input
   --> $DIR/malformed-derive-entry.rs:7:1
    |
 LL | #[derive]
-   | ^^^^^^^^^ help: missing traits to be derived: `#[derive(Trait1, Trait2, ...)]`
+   | ^^^^^^^^^ help: missing traits to be derived: ``#[derive(Trait1, Trait2, ...)]``
```

Comment:
we don't want double quotes here. can you track down the cause please?

Reply:
It was because I added backticks in the `.to_owned` function, which was already in charge of that. I've canceled the changes like these

Reply:
Ahh, I see.